src/signals-unix.c

```diff
@@ -120,6 +120,11 @@ static void jl_call_in_ctx(jl_ptls_t ptls, void (*fptr)(void), int sig, void *_c
     ctx->uc_mcontext.sp = rsp;
     ctx->uc_mcontext.regs[29] = 0; // Clear link register (x29)
     ctx->uc_mcontext.pc = (uintptr_t)fptr;
+#elif defined(_OS_FREEBSD_) && defined(_CPU_AARCH64_)
+    ucontext_t *ctx = (ucontext_t*)_ctx;
+    ctx->uc_mcontext.mc_gpregs.gp_sp = rsp;
+    ctx->uc_mcontext.mc_gpregs.gp_x[29] = 0; // Clear link register (x29)
+    ctx->uc_mcontext.mc_gpregs.gp_elr = (uintptr_t)fptr;
```

Comment:
That's a strange name for `pc`........

Reply:
Indeed.

https://lists.freebsd.org/pipermail/freebsd-arm/2017-February/015611.html

Reply:
OK, I kind of see where it gets its name from.... It's the address you return to after exception?... But seriously?...... Anyway, that's not important for the PR...